src/content/es/fundamentals/codelabs/your-first-pwapp/index.md

```diff
@@ -560,9 +566,9 @@ Para nuestra aplicación, esto proporciona una mejor experiencia que la red, rec
 
 ### Actualizar la lógica de la aplicación
 
-Como se mencionó anteriormente, la aplicación debe iniciar dos solicitudes asíncronas, una para el caché y otra para la red. La aplicación utiliza el objeto `caches` disponible en `window` para acceder a la memoria caché y recuperar los últimos datos. Este es un excelente ejemplo de mejora progresiva ya que el objeto `caches` puede no estar disponible en todos los navegadores, y si no es así, la solicitud de red debería funcionar.
+Como se mencionó anteriormente, la aplicación debe iniciar dos solicitudes asíncronas, una para la caché y otra para la red. La aplicación utiliza el objeto `caches` disponible en `window` para acceder a la memoria caché y recuperar los últimos datos. Este es un excelente ejemplo de mejora progresiva ya que el objeto `caches` puede no estar disponible en todos los navegadores, y si no es así, la solicitud de red debería funcionar.
 
-Actualice la función `getForecastFromCache()` para verificar si el objeto `caches` está disponible en el objeto global `window` y, si lo está, solicite los datos de la memoria caché.
+Actualiza la función `getForecastFromCache()` para verificar si el objeto `caches` está disponible en el objeto global `window` y, si lo está, solicite los datos de la memoria caché.
```

Comment:
`solicita los datos de la memoria caché.` to keep the conjugation in the second person.